assets/js/components/Layout/Layout.jsx

```diff
@@ -89,23 +89,24 @@ const Layout = () => {
                 onClick={handleSidebar}
                 title={isCollapsed ? 'Expand Sidebar' : 'Collapse Sidebar'}
               >
-                <EosKeyboardDoubleArrowLeft
-                  className="text-gray-400 hover:text-gray-300"
-                  flip={isCollapsed}
+                <EOS_KEYBOARD_DOUBLE_ARROW_LEFT
+                  size="24"
+                  color="currentColor"
```

Comment:
I'm not sure if this `currentColor` is needed, maybe it is

Reply:
I will look at what happens if I remove it. If its is unnecessary then I will remove it. It was only there initially as I saw it in an SVG implementation.

Reply:
@arbulu89 If I remove `currentColor` then it reverts to black and the classes don't have any effect. I think I will need to retain setting the `color` attribute to enable the classes to work too.